client/output.rs

```diff
@@ -232,27 +239,40 @@ pub fn print_log(task: &Task) {
         println!("End: {}", end.to_rfc2822());
     }
 
-    if let Some(stdout) = &task.stdout {
-        if !stdout.is_empty() {
-            println!(
-                "{}",
-                style("Std_out: ")
-                    .with(Color::Green)
-                    .attribute(Attribute::Bold)
-            );
-            println!("{}", stdout);
+    if !task_log.stdout.is_empty() {
+        if let Err(err) = print_task_output(&task_log, true) {
+            println!("Error while parsing stdout: {}", err);
         }
     }
 
-    if let Some(stderr) = &task.stderr {
-        if !stderr.is_empty() {
-            println!(
-                "{}",
-                style("Std_err: ")
-                    .with(Color::Red)
-                    .attribute(Attribute::Bold)
-            );
-            println!("{}", stderr);
-        }
+    if !task_log.stderr.is_empty() {
+        if let Err(err) = print_task_output(&task_log, false) {
+            println!("Error while parsing stderr: {}", err);
+        };
     }
 }
+
+/// Pritn the stdout of a finished process
+/// The logs are compressed using Brotli and then encoded to Base64
```

Comment:
Is it Brotli or Snappy?

Reply:
Snappy. Forgot to change the comment.

Brotli was way to slow